### PR TITLE
Fix bug in which the first transcript line is not highlighted.

### PR DESCRIPTION
**Pull request by amuamushu**
The first transcript line is never bolded because the styling of a transcript line is only updated when the current transcript line is updated to be the next line.
This PR fixes that issue by adding additional checks in `seekTranscript`

### CODE REVIEW

File: src/main/webapp/view/transcript/transcript.js [hunk 1]
```diff
@@ -112,7 +112,13 @@ function deleteTranscript() {
 /** Seeks transcript to {@code currentTime}, which is given in seconds. */
 function seekTranscript(currentTime) {
   const currentTimeMs = window.secondsToMilliseconds(currentTime);
-  if (currentTimeMs <= currentTranscriptLine.endTimestampMs) {
+  if (currentTimeMs < currentTranscriptLine.startTimestampMs) {
```
Comment (n-wach) [inline src/main/webapp/view/transcript/transcript.js:115]: Will `currentTranscriptLine` be `undefined` at first?

Comment (n-wach) [reply src/main/webapp/view/transcript/transcript.js:115]: .

Comment (amuamushu) [reply src/main/webapp/view/transcript/transcript.js:115]: I'm guessing it would be undefined if seekTranscript is called before the transcript finishes loading. 
But once the lines are added to the DOM, it shouldn't be undefined because currentTranscript is assigned when the lines are added to the DOM. 

https://github.com/googleinterns/zoomtube/blob/a8387b467ebd12eff34340f1d6349833d9be4053/src/main/webapp/view/transcript/transcript.js#L83

Comment (n-wach) [reply src/main/webapp/view/transcript/transcript.js:115]: Ok, thanks for adding an issue to track this

Comment (amuamushu) [reply src/main/webapp/view/transcript/transcript.js:115]: Of course, thanks for bring this up! For reference, the issue created is : #211

File: src/main/webapp/view/transcript/transcript.js [hunk 2]
```diff
@@ -112,7 +112,13 @@ function deleteTranscript() {
...
+    return;
+  }
+  if (isWithinCurrentTimeRange(currentTimeMs)) {
+    if (!isBolded(currentTranscriptLine)) {
+      addBold(currentTranscriptLine);
```
Comment (billie-cat) [inline src/main/webapp/view/transcript/transcript.js:120]: This addBold method could check if the line is already bold and return early if it is. The benefit is that you do not need to have nested if-statements.

File: src/main/webapp/view/transcript/transcript.js [hunk 3]
```diff
@@ -134,6 +140,20 @@ function removeBold(transcriptLineLiElement) {
   transcriptLineLiElement.classList.remove(BOLD_FONT_WEIGHT);
 }
 
+/** Checks if `transcriptLineLiElement` is bolded. */
```
Comment (billie-cat) [inline src/main/webapp/view/transcript/transcript.js:143]: nit: Reword this to: Returns `true` if ...

The statement: Checks does not imply it returns anything.

File: src/main/webapp/view/transcript/transcript.js [hunk 4]
```diff
@@ -134,6 +140,20 @@ function removeBold(transcriptLineLiElement) {
...
+function isBolded(transcriptLineLiElement) {
+  return transcriptLineLiElement.classList.contains(BOLD_FONT_WEIGHT);
+}
+
+/**
+ * Checks if `currentTimeMs` is within the time range for
```
Comment (billie-cat) [inline src/main/webapp/view/transcript/transcript.js:149]: here too

File: src/main/webapp/view/transcript/transcript.js (hunk 2 above)
Comment (amuamushu) [reply src/main/webapp/view/transcript/transcript.js]: done! I also added a similar check to `removeBold`.

File: src/main/webapp/view/transcript/transcript.js (hunk 3 above)
Comment (amuamushu) [reply src/main/webapp/view/transcript/transcript.js]: gotcha, done!

File: src/main/webapp/view/transcript/transcript.js (hunk 4 above)
Comment (amuamushu) [reply src/main/webapp/view/transcript/transcript.js]: done!

File: src/main/webapp/view/transcript/transcript.js [hunk 5]
```diff
@@ -124,16 +128,36 @@ function seekTranscript(currentTime) {
 
 /** Bolds the text in `transcriptLineLiElement` */
```
Comment (billie-cat) [inline src/main/webapp/view/transcript/transcript.js:129]: Update this comment to mention: Bolds the text if not already bold.

Comment (amuamushu) [reply src/main/webapp/view/transcript/transcript.js]: done, thanks for catching that!
